client/src/components/Utilities/GetCoordinates/GetCoordinates.tsx

```diff
@@ -252,16 +262,32 @@ const GetCoordinates: React.FC = () => {
               : null}
           </Table>
         );
-      } else if (error) {
-        return <Typography>{error}</Typography>;
+      } else if (responseWarnings?.length > 0) {
+        return <Typography>{responseWarnings}</Typography>;
       } else {
-        return <></>;
+        return (
+          <Typography>
+            An unknown error has occurred. Please{" "}
+            <Link href="https://github.com/cancervariants/fusion-curation/issues">
```

Comment:
I wonder if we should be linking to use the bug issue template

```suggestion
            <Link href="https://github.com/cancervariants/fusion-curation/issues/new?assignees=&labels=bug&projects=&template=bug-report.yaml">
```

Reply:
good idea!